src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,51 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import api from '../../api/api';
+
+// Actions
+const GET_ROCKETS = 'GET_ROCKETS';
+
+const initialState = {
+  rockets: [],
+  status: 'idle', // 'idle' | 'loading' | 'succeeded' | 'failed'
+  error: null,
+  msg: null,
+};
+
+// Thunks
+export const getRockets = createAsyncThunk(GET_ROCKETS, async () => {
+  try {
+    return await api.fetchRockets();
+  } catch (error) {
+    return error.message;
+  }
+});
+
+// Reducer
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder
+      .addCase(getRockets.pending, (state) => ({
+        ...state,
+        status: 'loading',
+      }))
+      .addCase(getRockets.fulfilled, (state, action) => ({
+        ...state,
+        rockets: action.payload,
+        status: 'succeeded',
+      }))
+      .addCase(getRockets.rejected, (state, action) => ({
+        ...state,
+        status: 'failed',
+        error: action.error.message,
```

Comment:
Similar to my previous recommendations. The `error.message`  is already passed to the action with the `payload` key from the catch return in line 19.

Using `error: action.payload` will automatically pass in the `error.message` value to the `error` key!